### PR TITLE
Dev epam language change

### PR DESCRIPTION
**Pull request by andrei-stoean**
Please review my code

### CODE REVIEW

**Review by AncaSava — APPROVED** [review]

File: src/test/java/epamcomautomation/languagechange/LanguageChangeTests.java [hunk 1]
```diff
@@ -1,6 +1,66 @@
...
+import static com.codeborne.selenide.Selenide.webdriver;
+import static com.codeborne.selenide.WebDriverConditions.url;
 
 public class LanguageChangeTests extends EpamBaseTest {
+
+    @Test
+    public void dropDownMenuAppears() {
+        var dropDownMenu = new HomePage()
+                .clickLanguageDropDown()
+                .checkLanguageDropDownVisible();
```
Comment (bonezoltan) [inline src/test/java/epamcomautomation/languagechange/LanguageChangeTests.java:18]: Where is the assert?

**Review by bonezoltan — COMMENTED** [review]
it is recommended to have 2 or more asserts when you verify if a page is loaded

File: src/test/java/epamcomautomation/languagechange/LanguageChangeTests.java (hunk 1 above)
Comment (Tamas-Demeny) [inline src/test/java/epamcomautomation/languagechange/LanguageChangeTests.java:18]: We need an assert, to be sure that the list is populated

**Review by Tamas-Demeny — CHANGES_REQUESTED** [review]
You have no assert for @Test dropdownMenuAppears()

File: src/test/java/epamcomautomation/languagechange/LanguageChangeTests.java (hunk 1 above)
Comment (andrei-stoean) [reply src/test/java/epamcomautomation/languagechange/LanguageChangeTests.java:18]: Added the assert

Comment (andrei-stoean) [reply src/test/java/epamcomautomation/languagechange/LanguageChangeTests.java:18]: Assert added

**Review by AncaSava — APPROVED** [review]

**Review by Tamas-Demeny — APPROVED** [review]
changes are approved

**Review by bonezoltan — APPROVED** [review]
look good

**Pull request merged**